src/mrpro/utils/kdata.py

```diff
@@ -0,0 +1,294 @@
+"""Utilities for kdata objects."""
+
+# Copyright 2023 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import copy
+from typing import Literal
+
+import torch
+from einops import rearrange
+from einops import repeat
+
+from mrpro.data import KData
+from mrpro.data import KTrajectory
+from mrpro.data import Limits
+from mrpro.utils import modify_acq_info
+
+
+def combine_k2_k1_into_k1(
+    kdata: KData,
+) -> KData:
+    """Reshape kdata from (... k2 k1 ...) to (... 1 (k2 k1) ...).
+
+    Parameters
+    ----------
+    kdata
+        K-space data (other coils k2 k1 k0)
+
+    Returns
+    -------
+        K-space data (other coils 1 (k2 k1) k0)
+    """
+
+    # Rearrange data
+    kdat = rearrange(kdata.data, 'other coils k2 k1 k0->other coils 1 (k2 k1) k0')
```

Comment:
I think it does not match the docstring.
This function will only work for a single others dimension.
Maybe we can use .... instead of other here?